docs/index.rst

```diff
@@ -47,6 +47,7 @@ often"* approach through automation. :ref:`Read more <about>`
    py/ui
    vhdl_libraries
    examples
+   py/vivado
```

Comment:
This doesn't feel like a top-level thing. I would prefer we have "Tool Integrations" at this level and the "Vivado Integration" one level down. While we don't strive to maintain a lot of tool integrations we can use this section to also describe things that are special to the various simulators we support.